src/components/pages/Offers/Offer/Confirmation/__specs__/Confirmation.spec.jsx

```diff
@@ -33,6 +33,33 @@ describe('confirmation page', () => {
     )
   })
 
+  it('should display the rights information when offer is PENDING', async () => {
```

Comment:
```suggestion
  it('should display the rights information when offer is pending', async () => {
```

Modifie le premier test pour ajouter que l'offre est en brouillon.

Reply:
:+1: